### DISCUSSION TITLE
How to add a new field to the model_has_role table

### DISCUSSION THREAD

**Discussion by BehroozBvk** [spatie/laravel-permission#1749] [Q&A]
Hi,
How to add a new field to the model_has_role table and override the assignRole method to fill in the new field.
In fact, I want to add a new Organization_id field to the model_has_role table and specify its value when assigning a role to the user.
I know how to add a field to migration, I just do not know how to implement it

Thanks for the tips along with the sample code.

**Comment by fredsal** [comment]
You can try https://github.com/spatie/laravel-permission/discussions/1782
